tests/zfs-tests/include/libtest.shlib

```diff
@@ -3620,7 +3620,7 @@ function get_arcstat # stat
 		kstat arcstats.$stat
 		;;
 	Linux)
-		kstat arcstats | awk "/$stat/"' { print $3 }'
+		kstat arcstats | awk "/^$stat	/"' { print $3 }'
```

Comment:
No objection, but did you hit a case where this was causing a problem?

Reply:
Yeah, see commit message.  The pattern passed in was matching both data_size and metadata_size, garbling the result.  Perhaps explains why the author didn't use get_arcstat in the first place?